src/main/java/com/factly/dega/domain/Factcheck.java

```diff
@@ -35,12 +35,14 @@
     @Field("introduction")
     private String introduction;
 
+    @NotNull
```

Comment:
@sravanoddiraju is it ok of its not null

Reply:
changed it to NotNull